files/default/finish_kore_config.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
```

Comment:
should the core.config actually be an erb file? It seems like that is the standard way to do chef rather than sed/awk?

Reply:
Good question! The thing is, we don't know at chef time what the correct IP address is going to be; that IP address is not assigned until machine boot time. That's why this has to run as part of cloud-init.